overview/What_Is_EPICS.rst

```diff
@@ -0,0 +1,56 @@
+What is EPICS?
+--------------
+
+.. image:: ../images/EPICS_black_blue_logo_rgb_small_v03.jpg
```

Comment:
I tried to build and this message wasn't found. Is that right ?

Reply:
Image also doesn't show up in the built website: https://epics--23.org.readthedocs.build/en/23/overview/What_Is_EPICS.html

Reply:
Should be found, I'm checking